app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted = command.split(" ")
+    if len(splitted) != 3:
+        return "Invalid input"
```

Comment:
The function should raise an exception for invalid input instead of returning a string. Consider using `raise ValueError('Invalid input')` to handle this case as per the task requirements.